Package.swift

```diff
@@ -21,7 +25,7 @@ let package = Package(
     .package(url: "https://github.com/ReactiveX/RxSwift.git", from: "5.0.0"),
   ],
   targets: [
-    .target(name: "MAVSDK-Swift",
+    .target(name: "Mavsdk",
```

Comment:
Shouldn't this target now depend on `mavsdk_server`? Could it be the reason why the build fails in CI?

Reply:
I just added it to try it but there may be an issue with binary targets as dependencies in other targets. We may have to create a separate Swift Package just for the binary dependency in another GitHub repo.

Reply:
Once a release is tagged on [MAVSDK-XCFramework](https://github.com/mavlink/MAVSDK-XCFramework) for v0.37.0 this should be ready to merge. It should be tested with the [MAVSDK-Swift-Example](https://github.com/mavlink/MAVSDK-Swift-Example) by adding [my fork](https://github.com/kylewludwig/MAVSDK-Swift) using branch [swiftpm-archiving-hotfix](https://github.com/kylewludwig/MAVSDK-Swift/tree/swiftpm-archiving-hotfix)